src/uu/chgrp/Cargo.toml

```diff
@@ -18,7 +18,7 @@ path = "src/chgrp.rs"
 clap = { version = "2.33", features = ["wrap_help"] }
 uucore = { version=">=0.0.9", package="uucore", path="../../uucore", features=["entries", "fs", "perms"] }
 uucore_procs = { version=">=0.0.6", package="uucore_procs", path="../../uucore_procs" }
-walkdir = "2.2"
+uu_chown = { version=">=0.0.6", package="uu_chown", path="../chown"}
```

Comment:
I guess it's just a matter of taste, but I would move duplicate code to `uucore`.

Reply:
Makes sense, I'll do that instead